infra/terraform/environments/dev/main.tf

```diff
@@ -57,6 +57,19 @@ module "user_nfs" {
     performance_mode = "maxIO"
 }
 
+module "user_nfs_softnas" {
+    source = "../../modules/user_nfs_softnas"
+
+    env = "${var.env}"
```

Comment:
OK, it's not worth changing anything then and as I said we're already covered by the default value 👍